README.md

```diff
@@ -216,10 +216,15 @@ method. When the token has expired a new set of tokens can be requested using
 Access tokens can be used to access resources on your resource server. Please refer to the document
 about [scope tokens](http://docs.telenordigital.com/connect/id/scope.html) for more details.
 
-### Retrieving the logged in user's ID
+### Retrieving information about the logged in user
+
+If you request email, phone, profile and/or name scope tokens and claims, you can access these
```

Comment:
"name" is only a claim, not a scope token. "profile" is only a scope token and not a claim. "email" and "phone" are both scope tokens and claims.

Maybe you can rewrite to something like the following?

"If you request user claims like email, phone, and name using either scope tokens or the claims parameter, (…). If both email and phone claims have been requested, we will also provided the username used for the authentication in the ID token.